code/__HELPERS/unsorted.dm

```diff
@@ -1159,5 +1159,9 @@ will handle it, but:
 	addtimer(CALLBACK(GLOBAL_PROC, /.proc/remove_images_from_clients, I, show_to), 0.5 SECONDS)
 
 
-/proc/send_global_signal(signal) //Wrapper for callbacks and the likes.
-	SEND_GLOBAL_SIGNAL(signal)
\ No newline at end of file
+/proc/send_global_signal(datum/signal, ...) //Wrapper for callbacks and the likes.
+	SEND_GLOBAL_SIGNAL(signal, args) //Passes the arguments as a list.
+
+
+/proc/send_signal(datum/source, signal, ...) //Wrapper for callbacks and the likes.
```

Comment:
All signals already pass their arguments properly, that's why components.dm contains the comments like this
`//from base of atom/Click(): (location, control, params, mob/user)`
This makes the entire PR invalid

Reply:
you're wrong, you can't make callbacks with macros.